Samples/HelloWorld/SiloTests/packages.config

```diff
@@ -1,10 +1,35 @@
-  <package id="Microsoft.Orleans.OrleansProviders" version="1.2.3" targetFramework="net451" />
-  <package id="Microsoft.Orleans.OrleansRuntime" version="1.2.3" targetFramework="net451" />
-  <package id="Microsoft.Orleans.TestingHost" version="1.2.3" targetFramework="net451" />
+  <package id="Microsoft.Extensions.Configuration" version="1.0.0" targetFramework="net451" />
+  <package id="Microsoft.Extensions.Configuration.Abstractions" version="1.0.0" targetFramework="net451" />
+  <package id="Microsoft.Extensions.Configuration.Binder" version="1.0.0" targetFramework="net451" />
+  <package id="Microsoft.Extensions.DependencyInjection" version="1.0.0" targetFramework="net451" />
+  <package id="Microsoft.Extensions.DependencyInjection.Abstractions" version="1.0.0" targetFramework="net451" />
+  <package id="Microsoft.Extensions.Primitives" version="1.0.0" targetFramework="net451" />
+  <package id="Microsoft.Orleans.Core" version="1.4.0" targetFramework="net451" />
+  <package id="Microsoft.Orleans.OrleansProviders" version="1.4.0" targetFramework="net451" />
+  <package id="Microsoft.Orleans.OrleansRuntime" version="1.4.0" targetFramework="net451" />
+  <package id="Microsoft.Orleans.TestingHost" version="1.4.0" targetFramework="net451" />
   <package id="Newtonsoft.Json" version="7.0.1" targetFramework="net451" />
+  <package id="System.Collections" version="4.0.11" targetFramework="net451" />
+  <package id="System.Collections.Concurrent" version="4.0.12" targetFramework="net451" />
+  <package id="System.Collections.Immutable" version="1.1.37" targetFramework="net451" />
+  <package id="System.ComponentModel" version="4.0.1" targetFramework="net451" />
+  <package id="System.ComponentModel.Primitives" version="4.1.0" targetFramework="net451" />
+  <package id="System.ComponentModel.TypeConverter" version="4.1.0" targetFramework="net451" />
+  <package id="System.Diagnostics.Debug" version="4.0.11" targetFramework="net451" />
+  <package id="System.Globalization" version="4.0.11" targetFramework="net451" />
+  <package id="System.IO" version="4.1.0" targetFramework="net451" />
+  <package id="System.Linq" version="4.1.0" targetFramework="net451" />
+  <package id="System.Linq.Expressions" version="4.1.0" targetFramework="net451" />
+  <package id="System.Reflection" version="4.1.0" targetFramework="net451" />
+  <package id="System.Reflection.Metadata" version="1.2.0" targetFramework="net451" />
+  <package id="System.Resources.ResourceManager" version="4.0.1" targetFramework="net451" />
+  <package id="System.Runtime" version="4.1.0" targetFramework="net451" />
+  <package id="System.Runtime.Extensions" version="4.1.0" targetFramework="net451" />
+  <package id="System.Runtime.InteropServices" version="4.1.0" targetFramework="net451" />
+  <package id="System.Threading" version="4.0.11" targetFramework="net451" />
+  <package id="System.Threading.Tasks" version="4.0.11" targetFramework="net451" />
```

Comment:
are those updated by visual studio tooling? this is a lot of new packages..
otherwise looks good to me